src/coreclr/vm/gcenv.ee.cpp

```diff
@@ -1651,6 +1651,19 @@ void GCToEEInterface::AnalyzeSurvivorsFinished(size_t gcIndex, int condemnedGene
             s_forcedGCInProgress = false;
             reportGenerationBounds();
             FireEtwGenAwareEnd((int)gcIndex, GetClrInstanceId());
+            if (gcGenAnalysisDump)
+            {
+                EX_TRY
+                {
+#ifdef HOST_WIN32
```

Comment:
I have not seen this define used in the runtime code except in one file that I believe came from mono folks. the convention in the coreclr runtime code is to use 

```
#ifdef TARGET_UNIX
    PAL_Call();
#else 
    Windows_Call();
#endif //TARGET_UNIX
```
probably better to go with the convention.